tsdb/engine/tsm1/compact.go

```diff
@@ -253,8 +253,8 @@ func (c *DefaultPlanner) ParseFileName(path string) (int, int, error) {
 	return c.FileStore.ParseFileName(path)
 }
 
-// FullyCompacted returns true if the shard is fully compacted.
-func (c *DefaultPlanner) FullyCompacted() (bool, string) {
+// CompactionOptimizationAvailable returns true if the shard is fully compacted.
```

Comment:
What does a `true` return mean?  Optimization is available, or not available?

Would this be `CompactionOptimizationNotAvailable`?

Reply:
Sorry--yes you are correct true would mean there are no optimizations. Let me change that.